seed/ts-sdk/version/src/Client.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * This file was auto-generated by Fern from our API Definition.
+ */
+
+import * as core from "./core";
+import * as version from "./api/version";
+import { User } from "./api/resources/user/client/Client";
+
+export declare namespace SeedVersionClient {
+    interface Options {
+        environment: core.Supplier<string>;
+        /** Override the X-API-Version header */
+        xApiVersion?: version.SeedVersionVersion;
+    }
+
+    interface RequestOptions {
+        /** The maximum time to wait for a response in seconds. */
+        timeoutInSeconds?: number;
+        /** The number of times to retry the request. Defaults to 2. */
+        maxRetries?: number;
+        /** A hook to abort the request. */
+        abortSignal?: AbortSignal;
+        /** Override the X-API-Version header */
+        xApiVersion?: version.SeedVersionVersion;
```

Comment:
can we inline the version right here ?

Reply:
`"1.0.0" | "2.0.0" | "latest"`

Reply:
Discussed offline - I originally defined a separate type to keep things DRY in the generated code, but this ends up hindering the autocomplete experience. With this, I have to jump to definition to see the available set of versions rather than just seeing them on hover.

We'll go with the in-lined version approach, but continue to generate the `version.ts` file at the top level so users can easily see what versions are available (similar to our story with `environments.ts` and default environment).